ChameleonForms/FieldGenerators/Handlers/DateTimeHandler.cs

```diff
@@ -36,7 +37,15 @@ public override IHtmlString GenerateFieldHtml(IReadonlyFieldConfiguration fieldC
         public override void PrepareFieldConfiguration(IFieldConfiguration fieldConfiguration)
         {
             if (!string.IsNullOrEmpty(FieldGenerator.Metadata.DisplayFormatString))
-                fieldConfiguration.Attr("data-val-format", FieldGenerator.Metadata.DisplayFormatString.Replace("{0:", "").Replace("}", ""));
+            {

```

Comment:
Unit tests.

There are plenty of examples of approval tests in there - have you used Approval tests before?